tutorials/fundamentals/trim0.md

```diff
@@ -0,0 +1,42 @@
+To remove leading zeroes from a number string in JavaScript, you can simply multiply the string by 1.
+
+```javascript
+let x = '0042';
+x = x * 1;
```

Comment:
The problem here is that you're converting the string to a number first. Which is nice, but what happens if you want to trim leading zeros from a string that JavaScript can't convert to a number? I'd recommend we put the RegExp approach first, and then add the `parseInt()` approach.